translator/tocwconfig/sampleConfig/appsignals_and_eks_config.yaml

```diff
@@ -353,6 +353,7 @@ processors:
         detectors:
             - eks
             - env
+            - ecs
```

Comment:
Why is this getting added to the EKS configuration?

Reply:
Added a new resourcedetection template for ecs.